backend/canisters/user/impl/src/timer_job_types.rs

```diff
@@ -44,6 +56,19 @@ pub struct MessageReminderJob {
     pub reminder_created_message_index: MessageIndex,
 }
 
+impl From<MessageReminderJobPrevious> for MessageReminderJob {
+    fn from(value: MessageReminderJobPrevious) -> Self {
+        MessageReminderJob {
+            reminder_id: value.reminder_id,
+            chat: Chat::Group(value.chat_id),
```

Comment:
It could be a DirectChat... so maybe we keep the previous job types for a week or so until all the old ones are processed.

Reply:
Oh actually we can detect if it is a direct chat because we hold all the direct chats in this canister